bin/gsign

```diff
@@ -85,4 +95,9 @@ assert_path = File.join(release_path, "#{package_name}-build.assert")
 File.open(assert_path, "w") do |io|
   io.write result.to_yaml
 end
-system!("gpg --detach-sign -u \"#{signer}\" \"#{assert_path}\"")
+if @options[:ecdsa]
+  ecdsa_path = File.join(@options[:armory], release_scripts, "signassert.py")
```

Comment:
Ooops, release_scripts should be in quotation marks. I'll fix that with next commit.